multiqc/modules/longranger/longranger.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Longranger """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+import os
+import csv
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Longranger module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Longranger', anchor='longranger',
+        href="https://www.10xgenomics.com/",
+        info="A set of analysis pipelines that perform sample demultiplexing, "
+        "barcode processing, alignment, quality control, variant calling, phasing, "
+        "and structural variant calling.")
+
+        self.headers = OrderedDict()
+        self.headers['longranger_version'] = {
+                'scale': False,
+                'title': 'Version',
+                'description': 'The version of the Long Ranger software used to generate the results.',
+                'hidden': True
+        }
+        self.headers['instrument_ids'] = {
+                'scale': False,
+                'title': 'Instrument ID',
+                'description': 'The list of instrument IDs used to generate the input reads.',
+                'hidden': True
+        }
+        self.headers['gems_detected'] = {
+                'scale': 'RdYlGn',
+                'title': '# gems',
+                'modify': lambda x: float(x) / 1000000.0,
+                'suffix': 'M',
+                'description': 'The number of Chromium GEMs that were collected and which generated a non-trivial number of read-pairs.'
+        }
+        self.headers['mean_dna_per_gem'] = {
+                'scale': 'RdYlGn',
+                'title': 'DNA per gem',
+                'modify': lambda x: float(x) / 1000000.0,
+                'suffix': 'M',
+                'description': 'The average number of base pairs of genomic DNA loaded into each GEM. This metric is based on the observed extents of read-pairs on each molecule.',
+                'hidden': True
+        }
+        self.headers['bc_on_whitelist'] = {
+                'scale': 'RdYlGn',
```

Comment:
Please avoid using the same colour scheme for everything. It's best if no two adjacent columns share the same scheme, it makes the table much easier to read and interpret.

Reply:
I also use `PuBu` and `YlGn` ;)
Then I would just alternate the columns. `YlGn`, `PuBu`, `YlGn`, ...

Reply:
But you have [so many to choose from](http://multiqc.info/docs/#table-colour-scales-1)! 😉

Reply:
![colour bomb](https://i.ytimg.com/vi/VBwrMYZcwiM/hqdefault.jpg)

Reply:
I had to [look that one up](https://www.youtube.com/watch?v=T9MAmWnOznI) 😉